src/api/artifact/abstractor/resolver/chart/chart.go

```diff
@@ -31,16 +33,17 @@ const (
 	// ArtifactTypeChart defines the artifact type for helm chart
 	ArtifactTypeChart        = "CHART"
 	AdditionTypeValues       = "VALUES.YAML"
-	AdditionTypeReadme       = "README"
+	AdditionTypeReadme       = "README.MD"
```

Comment:
Please update the swagger file if change it to `README.MD`: https://github.com/goharbor/harbor/blob/master/api/v2.0/swagger.yaml#L275